frontend/src/Pages/AdminManagement.tsx

```diff
@@ -168,107 +168,102 @@ export default function AdminManagement() {
                         data-tip="Add Administrator"
                     >
                         <button
-                            className="btn btn-primary btn-sm"
+                            className="btn btn-primary btn-sm text-white" // Makes the text white
                             onClick={() => addUserModal.current?.showModal()}
                         >
-                            <UserPlusIcon className="h-4" />
+                            <PlusIcon className="h-4 border border-white rounded-full p-1 mr-2" />{' '}
+                            {/* Circular border around PlusIcon */}
+                            Add Admin
                         </button>
                     </div>
                 </div>
-                <table className="table">
-                    <thead>
-                        <tr className="border-gray-600">
-                            <th className="flex">
-                                <span>Name</span>
-                            </th>
-                            <th>Username</th>
-                            <th>Last Updated</th>
-                            <th>Actions</th>
-                        </tr>
-                    </thead>
-                    <tbody>
-                        {!isLoading &&
-                            !error &&
-                            userData.map((user: User) => {
-                                const updatedAt = new Date(user.updated_at);
-                                return (
-                                    <tr
-                                        key={user.id}
-                                        className="border-gray-600"
-                                    >
-                                        <td>
-                                            {user.name_first} {user.name_last}
-                                        </td>
-                                        <td>{user.username}</td>
-                                        <td>
+                <div className="w-full space-y-4">
```

Comment:
@MillerSenior Should this be a `table` instead of `div` tags? The issue says to switch to the `table-2` style class seen within `app.css`. I think we need it to be a table so we can center the column headers like the Figma shows.

Reply:
Done